samples/delay/Makefile

```diff
@@ -18,17 +18,23 @@
 # PROFITS, BUSINESS INTERRUPTION, LOSS OF PROGRAMS OR OTHER DATA   
 # ON YOUR INFORMATION HANDLING SYSTEM OR OTHERWISE.                
 #                                                                  
-# (C) Copyright IBM Corp. 2017  All Rights reserved.         
+# (C) Copyright IBM Corp. 2019  All Rights reserved.         
```

Comment:
Don't remove the earliest copyright date, instead add the latest new one, e.g. 2017-2019

Reply:
agree